PED/SetPedMinGroundTimeForStungun.md

```diff
@@ -8,11 +8,84 @@ ns: PED
 void SET_PED_MIN_GROUND_TIME_FOR_STUNGUN(Ped ped, int ms);
 ```
 
+Overwrites the minimum time the ped will stay on the ground for after being stunned. Setting this while the ped is stunned will not alter the duration of the current stun (but will still effect future stunns).
+
+Unlike what this native's name suggests, it also effects the time for other "weapons", such as WEAPON_ELECTRIC_FENCE.
```

Comment:
```suggestion
**NOTE**: Unlike what the native name suggestios, this works on any weapon that has its `DamageType` in the `weapons.meta` set to `ELECTRIC`.
```
I need to verify this, just an guess currently

Reply:
I'll leave this unresolved until it is confirmed. I'll also take a look at it sometime this afternoon.

Reply:
Okay, after some testing, it seems correct that this native affects all damage from weapons where the `DamageType` is set to `ELECTRIC`. I have tried this with `WEAPON_ELECTRIC_FENCE`, `WEAPON_STUNROD`, `WEAPON_STUNGUN` and `WEAPON_STUNGUN_MP`. Custom (add-on) weapons also behaved the same when the `DamageType` was set to `ELECTRIC`. I could not find a way to test `VEHICLE_WEAPON_PLAYER_LASER` and `VEHICLE_WEAPON_ENEMY_LASER`, but I presume those are just leftovers from R*.

I am pretty confident that your suggested change is indeed correct, so I am going to commit it. If you discover something that contradicts these findings, please let me know.